### PR TITLE
Kate yoak/sales enablement

### PR DESCRIPTION
**Pull request by KateYoak**
### What does this PR do?
Adding a sales enablement page for a new partners section of the docs

### Motivation
Developing a standard set of guidance docs for partners

https://docs-staging.datadoghq.com/KateYoak/sales-enablement/partners/sales-enablement

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/partners/sales-enablement.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: Partner Sales Enablement Guide
+kind: documentation
+description: "A training roadmap for a new MSP to be ready to lead a Datadog sales engineering process"
+private: true
+---
+## Datadog basics
+
+First head over to the <a href="https://partners.datadoghq.com/" target="_blank">Partner Portal</a>, log in and click to Training. From there, click to Get Started with the Sales Certification.
```
Comment (hestonhoffman) [inline content/en/partners/sales-enablement.md:9]:
```suggestion
First, head over to the <a href="https://partners.datadoghq.com/" target="_blank">Partner Portal</a>. Log in and click **Training**. Under Sales Certification, click **Get started today**.
```

File: content/en/partners/sales-enablement.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+{{<img src="partners/portal_sales_go.png" alt="Partner Portal Sales Certification" style="max-width:734px">}}
+
+This course walks you through the basics you need to work with your opportunity.
+
+This course helps you to:
```
Comment (hestonhoffman) [inline content/en/partners/sales-enablement.md:15]:
```suggestion
The course helps you to:
```

File: content/en/partners/sales-enablement.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    -  Intro to differentiators
+    -  7 signs of technical fit
+
+- Get ready to articulate the message to different audiences with:
+    -  Customers and Personas
+    -  Elevator pitch training
+
+## Technical demo
+
+Once you have the idea of the Datadog basics, you should get your Demo Certification.
```
Comment (hestonhoffman) [inline content/en/partners/sales-enablement.md:32]:
```suggestion
Once you're familiar with the Datadog basics, you should get your Demo Certification.
```
Is the Demo Certification part of the Sales Certification? Or are you telling the reader to do the Demo Certification after they've completed the Sales Certification? If it's a separate certification that you complete after you're done with the sales certification, I would say this:
> Once you've completed the Sales Certification and you're familiar with the Datadog basics, you should get your Demo Certification.

File: content/en/partners/sales-enablement.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+{{<img src="partners/portal_demo_go.png" alt="Partner Portal Demo Certification" style="max-width: 367px">}}
+
+Not only does this course teach you to deliver a technical demo, it further helps you understand Datadog's approach to observability and its unified collaborative platform.
+
+## Proof of value
+
+After giving a technical demo, you will be working with the customer to develop an initial proof of value solution. To prepare for this, obtain the Datadog Technical Certification.
```
Comment (hestonhoffman) [inline content/en/partners/sales-enablement.md:40]:
```suggestion
After giving a technical demo, you'll work with the customer to develop an initial proof of value solution. To prepare for this, obtain the Datadog Technical Certification.
```

File: content/en/partners/sales-enablement.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+{{<img src="partners/portal_tech_go.png" alt="Partner Portal Technical Certification" style="max-width:367px">}}
+
+This course
```
Comment (hestonhoffman) [inline content/en/partners/sales-enablement.md:44]:
```suggestion
This course:
```

File: content/en/partners/sales-enablement.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+- Explains the basics behind Datadog core products
+- Addresses the market positioning of each
```
Comment (hestonhoffman) [inline content/en/partners/sales-enablement.md:46]:
```suggestion
- Addresses the market positioning of each product
```

File: content/en/partners/sales-enablement.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+- Describes customer success 
+- Helps you position individual product offerings
+
+To guide a customer through a successful proof of value engagement, you need deeper product knowledge. Get started at the Datadog Learning Center with [Datadog Fundamentals Learning Path][1]. The courses include hands-on labs and get you a technical understanding of how Datadog works.
```
Comment (hestonhoffman) [inline content/en/partners/sales-enablement.md:50]:
```suggestion
To guide a customer through a successful proof of value engagement, you need deeper product knowledge. Get started with the [Datadog Fundamentals Learning Path][1] offered by the Datadog Learning Center. The Learning Center courses include hands-on labs and deepen your technical understanding of how Datadog works.
```

File: content/en/partners/sales-enablement.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+
+## Pricing
+
+In order to complete your quote, you need pricing basics.
+
+- [Datadog pricing documentation][2] describes how pricing works: units of measure and metering approach at Datadog
+- [Customer plans][3] provides detail into standard end-customer pricing options
+- Finally, to understand discounts offered, check out the Partner Portal pricing tab:
+
+{{<img src="partners/portal_pricing_go.png" alt="Partner Portal Pricing Info" style="max-width:734px">}}
```
Comment (hestonhoffman) [inline content/en/partners/sales-enablement.md:60]:
```suggestion
- [Datadog's pricing][2] describes how pricing works, including the units of measurement and the metering approach at Datadog.
- [Customer plans][3] provides details about standard end-customer pricing options.
- Finally, to understand discounts offered, check out the Partner Portal pricing tab:

{{<img src="partners/portal_pricing_go.png" alt="Partner Portal Pricing Info" style="max-width:734px">}}
```

File: content/en/partners/sales-enablement.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+
+## The Datadog sales engineering process
+
+If you have gone through the recommended training outlined above, you are now ready to go conduct the Datadog sales engineering process:
```
Comment (hestonhoffman) [inline content/en/partners/sales-enablement.md:64]:
```suggestion
If you have gone through the recommended training outlined above, you're ready to conduct the Datadog sales engineering process:
```

File: content/en/partners/sales-enablement.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+- Qualify Opportunity
+- Register Opportunity
+- Technical Demo
+- Proof of Value
+- Configure, Price, Quote
+
+Should you have any questions during your journey, contact partner-support@datadoghq.com 
+
+[1]: https://learn.datadoghq.com/bundles/dd-fundamentals
+[2]: https://docs.datadoghq.com/account_management/billing/pricing/
```
Comment (hestonhoffman) [inline content/en/partners/sales-enablement.md:76]:
```suggestion
[2]: /account_management/billing/pricing/
```

File: content/en/partners/sales-enablement.md [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+    -  Intro to differentiators
+    -  7 signs of technical fit
+
+- Get ready to articulate the message to different audiences with:
+    -  Customers and Personas
+    -  Elevator pitch training
+
+## Technical demo
+
+Once you're familiar with the Datadog basics, you should get your Demo Certification.
```
Comment (KateYoak) [inline content/en/partners/sales-enablement.md:32]:
```suggestion
Once you have completed the Datadog Sales Certification and are familiar with the Datadog basics, you should get your Demo Certification.
```

**Review by hestonhoffman — APPROVED** [review]

**Pull request merged**